src/containers/home/HomeContainer.tsx

```diff
@@ -3,12 +3,14 @@ import { useSelector } from "react-redux";
 
 import { RootState } from "../../store/reducers";
 import { useAppDispatch } from "store/store";
-import { useGetBaseQuery } from "./api";
+import { useGetCountryQuery } from "./api";
 import { updateValue } from "./slice";
 import { Button } from "components";
 
 const HomeContainer: React.FC = () => {
```

Comment:
Lets simply name it Home instead of HomeContainer?

Reply:
Seeing multiple instances of Home containers arising, hence changing the container name from 'Home Container' to 'Home' and the page name from 'Home' to 'Homepage'